docs/getting-started/ios/fastlane-swift.md

```diff
@@ -42,6 +42,59 @@ Run `fastlane <laneName>` in your terminal to execute _fastlane_.
 
 🎉
 
+## Get Started (SPM) (Beta)
+
+Swift Package Manager support is included as a preliminary feature that allows the usage of _fastlane_ with other SPM packages.
+
+### Step 1
+
+Create an executable Swift Package Manager project with `swift package init --type executable`.
+
+### Step 2
+
+Add the _fastlane_ dependency to your `Package.swift`.
+
+```swift
+.package(name: "Fastlane", url: "https://github.com/fastlane/fastlane", .from("2.179.0"))
```

Comment:
Ahh, I just tried to add this line to my own `Package.swift` file and it said it was invalid. Here's what I think the correct syntax needs to be.

```suggestion
.package(name: "Fastlane", url: "https://github.com/fastlane/fastlane", from: "2.179.0")
```

Hope that helps! :)

Reply:
Yep, I meant `.upToNextMajor(from: "2.179.0")` but it really does not matter, what a day I had🤯